### PR TITLE
test(news): add integration tests for news and cleaned up news a bit

### LINKED ISSUE #113: Skriv tester for viktigste funksjonalitet
Skrive enhetstester for:
- [x] Event
- [ ] News
- [x] Registration

Integrasjonstester:
- [x] Event
- [ ] News
- [x] Registration

### PR DESCRIPTION
**Pull request by Eirsteir**
## Added integration tests for news endpoint and refactor news model, serializer and view a little to meet our current standards
Link to Trello task: closes #113 

Comments/issues:
I have not written any unittests, since there was little to no custom methods and logic for news.

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] PR name is descriptive
- [x] The PR includes the link to the relevant Trello task
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been reviewed and added / updated if needed (for bug fixes / features)

- [x] `python manage.py runserver` was run locally witout errors

## Pull request type

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

### CODE REVIEW

**Review by norgor — COMMENTED** [review]
Looks good overall! I just commented a few small things 🥇

File: app/tests/test_news_integration.py [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+    response = client.post(_get_news_url(), news_post_data)
+
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    ("group_name", "expected_status_code"),
+    [
+        (AdminGroup.HS, 201),
```
Comment (Zenjjim) [inline app/tests/test_news_integration.py:221]: use status.HTTP

File: app/tests/test_news_integration.py [hunk 2]
```diff
@@ -0,0 +1,297 @@
...
+    response = client.delete(url)
+
+    assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.django_db
+@pytest.mark.parametrize(
+    ("group_name", "expected_status_code"),
+    [
+        (AdminGroup.HS, 200),
```
Comment (Zenjjim) [inline app/tests/test_news_integration.py:274]: use status.HTTP

**Review by Zenjjim — APPROVED** [review]

**Review by norgor — CHANGES_REQUESTED** [review]
Good stuff! 🍔

**Review by norgor — APPROVED** [review]
Sorry, I clicked wrong again... Good job! 😛

**Pull request merged**